src/deepsparse/server/build_logger.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Specifies the mapping from the ServerConfig to the DeepSparse Logger
+"""
+
+from typing import Any, Dict, List, Union
+
+from deepsparse import loggers as logger_objects
```

Comment:
nit: prefer to explicitly import classes and functions from this module rather than the entire module (logger_objects also could be a misnomer as non logger objects may be imported as well)